lib/modules/manager/sbt/extract.ts

```diff
@@ -36,6 +38,11 @@ interface Ctx {
 
 const scala = lang.createLang('scala');
 
+const sbtVersionRegex = regEx(
+  '^[ ]*sbt\\.version[ ]*=[ ]*(?<version>\\d+\\.\\d+\\.\\d+)[ ]*$',
```

Comment:
are you sure spaces are allowed before the config item?
```suggestion
  '(?:^|\\n)sbt\\.version *= *(?<version>\\d+\\.\\d+\\.\\d+)(?:\\s|$)',
```
i would use something like that

Reply:
I verified and you're right, spaces are actually only allowed around the `=` sign. At the beginning or the end, it will be "tolerated" but generates warning from sbt, we should not try to support these as sbt rewrite the file anyway in these cases.

Reply:
I'm not an expert with regex but if I use this one, the matched string contains the non capturing groups as well which is not what we want. In the end, it seems to me we just don't need to use the `^` and `$` in the regex if we are matching the whole content and not line by line.

Reply:
I just used `'sbt\\.version *= *(?<version>\\d+\\.\\d+\\.\\d+)'` in the end.